### PR TITLE
Coloring / Styling Pass on Editor

### PR DESCRIPTION
**Pull request by Fignewton5**
## First rough styling / coloring pass to make editor look more like VSCode.

- Editor feels much less like a carnival
- Added colors file
- Also fixed references to non-existent galmor css objects that were causing errors when loading the `CardEditor`

botstudio_dark1
botstudio_dark

### CODE REVIEW

File: client-v2/src/ui/editor/cardEditor/cardPreviewWindow.js [hunk 1]
```diff
@@ -50,23 +51,26 @@ const CSS = css({
...
         overflow: 'auto',
         height: 'calc(100% - 24px)',
-        width: '100%'
+        width: '100%',
+        backgroundColor: Colors.PANEL_BACKGROUND_DARK,
+        color: Colors.PANEL_FOREGROUND_DARK
+    },
+
+    ' .ac-container > div': {
+        color: `${Colors.PANEL_FOREGROUND_DARK} !important`
```
Comment (compulim) [inline client-v2/src/ui/editor/cardEditor/cardPreviewWindow.js]: I think we couldn't do it without `!important`.

Comment (Fignewton5) [reply client-v2/src/ui/editor/cardEditor/cardPreviewWindow.js]: Yeah, there's no real way to get the dark theme to work with the Adaptive Cards renderer without overriding their styles.

File: client-v2/src/ui/layout/expandCollapse.js [hunk 2]
```diff
@@ -36,14 +36,19 @@ import PropTypes from 'prop-types';
...
-        backgroundColor: 'hotpink',
+        backgroundColor: Colors.SECTION_HEADER_BACKGROUND_DARK,
+        color: Colors.SECTION_HEADER_FOREGROUND_DARK,
+        fontFamily: 'Segoe UI Semibold',
+        textTransform: 'uppercase',
         display: 'flex',
         lineHeight: '30px',
 
         '> .content': {
+            paddingLeft: '8px',
```
Comment (compulim) [inline client-v2/src/ui/layout/expandCollapse.js]: Number could be used here.

File: client-v2/src/ui/layout/expandCollapse.js [hunk 3]
```diff
@@ -36,14 +36,19 @@ import PropTypes from 'prop-types';
...
 
 import { filterChildren } from '../utils';
+import * as Colors from '../colors/colors';
 
 const CSS = css({
     '> header': {
-        backgroundColor: 'hotpink',
+        backgroundColor: Colors.SECTION_HEADER_BACKGROUND_DARK,
+        color: Colors.SECTION_HEADER_FOREGROUND_DARK,
+        fontFamily: 'Segoe UI Semibold',
```
Comment (compulim) [inline client-v2/src/ui/layout/expandCollapse.js]: Do we need to specify fallback for Mac?

File: client-v2/src/ui/shell/explorer/conversationExplorer.js [hunk 4]
```diff
@@ -41,23 +41,30 @@ import * as ConversationActions from '../../../data/action/conversationActions';
...
     display: 'flex',
     flexDirection: 'column',
     listStyleType: 'none',
     margin: 0,
-    padding: 0
+    padding: 0,
+
+    ' > li': {
+        padding: '4px 24px',
+        fontFamily: 'Segoe UI'
```
Comment (compulim) [inline client-v2/src/ui/shell/explorer/conversationExplorer.js]: Do we need fallback like, `fontFamily: '\'Segoe UI\', \'Helvetica Neue\', Arial, sans-serif'`?

File: client-v2/src/ui/shell/explorer/explorerSet.js [hunk 5]
```diff
@@ -44,7 +46,11 @@ const CSS = css({
 });
 
 const HEADER_CSS = css({
-    lineHeight: '30px'
+    lineHeight: '30px',
+    color: Colors.EXPLORER_FOREGROUND_DARK,
+    paddingLeft: '16px',
+    textTransform: 'uppercase',
+    fontFamily: 'Segoe UI'
```
Comment (compulim) [inline client-v2/src/ui/shell/explorer/explorerSet.js]: Fallback?

File: client-v2/src/ui/shell/explorer/explorerSet.js [hunk 6]
```diff
@@ -44,7 +46,11 @@ const CSS = css({
 });
 
 const HEADER_CSS = css({
-    lineHeight: '30px'
+    lineHeight: '30px',
+    color: Colors.EXPLORER_FOREGROUND_DARK,
+    paddingLeft: '16px',
```
Comment (compulim) [inline client-v2/src/ui/shell/explorer/explorerSet.js]: Could use `number` here.

File: client-v2/src/ui/shell/explorer/folderNotOpenExplorer.js [hunk 7]
```diff
@@ -76,8 +83,10 @@ class FolderNotOpenExplorer extends React.Component {
                         title="No Folder Opened"
                     >
                         <ExpandCollapseContent>
-                            <div>You have not yet opened a folder.</div>
-                            <button onClick={ this.handleOpenFolderClick }>Open Folder</button>
+                            <div className='folder-empty-state'>
```
Comment (compulim) [inline client-v2/src/ui/shell/explorer/folderNotOpenExplorer.js]: Double-quote on HTML/JSX side.

File: client-v2/src/ui/shell/explorer/formExplorer.js [hunk 8]
```diff
@@ -35,23 +35,30 @@ import { css } from 'glamor';
...
 
 const BOTS_CSS = css({
     display: 'flex',
     flexDirection: 'column',
     listStyleType: 'none',
     margin: 0,
-    padding: 0
+    padding: 0,
+
+    ' > li': {
```
Comment (compulim) [inline client-v2/src/ui/shell/explorer/formExplorer.js]: Please eliminate the prefixing space. If it isn't working, try `'& > li'`.

File: client-v2/src/ui/shell/explorer/luisExplorer.js [hunk 9]
```diff
@@ -35,23 +35,30 @@ import { css } from 'glamor';
...
 
 const BOTS_CSS = css({
     display: 'flex',
     flexDirection: 'column',
     listStyleType: 'none',
     margin: 0,
-    padding: 0
+    padding: 0,
+
+    ' > li': {
```
Comment (compulim) [inline client-v2/src/ui/shell/explorer/luisExplorer.js]: Prefixing space.

File: client-v2/src/ui/shell/explorer/luisExplorer.js [hunk 10]
```diff
@@ -35,23 +35,30 @@ import { css } from 'glamor';
...
+        padding: '4px 24px',
+        fontFamily: 'Segoe UI'
```
Comment (compulim) [inline client-v2/src/ui/shell/explorer/luisExplorer.js]: Fallback?

File: client-v2/src/ui/shell/explorer/formExplorer.js [hunk 11]
```diff
@@ -35,23 +35,30 @@ import { css } from 'glamor';
...
+        padding: '4px 24px',
+        fontFamily: 'Segoe UI'
```
Comment (compulim) [inline client-v2/src/ui/shell/explorer/formExplorer.js]: Fallback?

File: client-v2/src/ui/shell/explorer/qnaExplorer.js [hunk 12]
```diff
@@ -35,23 +35,30 @@ import { css } from 'glamor';
...
 
 const BOTS_CSS = css({
     display: 'flex',
     flexDirection: 'column',
     listStyleType: 'none',
     margin: 0,
-    padding: 0
+    padding: 0,
+
+    ' > li': {
```
Comment (compulim) [inline client-v2/src/ui/shell/explorer/qnaExplorer.js]: Prefixing space.

File: client-v2/src/ui/shell/explorer/qnaExplorer.js [hunk 13]
```diff
@@ -35,23 +35,30 @@ import { css } from 'glamor';
...
+        padding: '4px 24px',
+        fontFamily: 'Segoe UI'
```
Comment (compulim) [inline client-v2/src/ui/shell/explorer/qnaExplorer.js]: Fallback?

File: client-v2/src/ui/shell/explorer/explorerSet.js (hunk 6 above)
Comment (Fignewton5) [reply client-v2/src/ui/shell/explorer/explorerSet.js]: So I found out that a lineheight of `30` and a lineheight of `30px` result in two very different things haha. 30 is HUGE

**Pull request merged**